release/release_tests.yaml

```diff
@@ -3288,7 +3288,7 @@
     frequency: multi
     cluster:
       app_config: ray_sgd_training_app.yaml
-      compute_template: ray_sgd_training_smoke_compute.yaml
+      cluster_compute: ray_sgd_training_smoke_compute.yaml
```

Comment:
Note that this smoke test is never ran (because the main test is also ran with `multi` frequency)